katello/katello-change-hostname

```diff
@@ -67,6 +67,10 @@ opt_parser = OptionParser.new do |opt|
     end
   end
 
+  opt.on("-y", "--assumeyes", "Bypass interaction by answering yes") do |confirm|
```

Comment:
For consistency with katello-restore, recommend using 'Answer yes for all questions'.